functions/utilities.zsh

```diff
@@ -111,6 +111,15 @@ case $(uname) in
       ;;
 esac
 
+# Not all OSes support the '-o' parameter
+# That's why this second condition is needed 
+case $(uname -o 2>/dev/null) in
```

Comment:
Thanks for your kind words! Yes, you got that right and I've fixed it. Let me know what you think now!